web_interface/blueprints/api_v3.py

```
@@ -120,6 +120,27 @@ def _get_plugin_version(plugin_id: str) -> str:
+    version is *ahead* of the published registry is not flagged as needing an
+    update. If either version string can't be parsed, falls back to a plain
+    inequality check (any difference is surfaced so the user can reconcile).
+    """
+    if not installed_version or not latest_version:
+        return False
+    if installed_version == latest_version:
+        return False
+    try:
+        from packaging.version import parse as _parse_version
+        return _parse_version(latest_version) > _parse_version(installed_version)
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)